ts/ui/menu/Menu.ts

```diff
@@ -981,6 +974,29 @@ export class Menu {
         };
     }
 
+    /**
+     * Create JSON for an a11y specific variable.
+     *
+     * @param {keyof MenuSettings} name   The setting for which to make a variable
+     * @param {(T) => void} action        Optional function to perform after setting the value
```

Comment:
This argument doesn't seem to be there any longer.

Reply:
Mindless copy and pasting...